### PR TITLE
feat: enable consistency options support by default & disable associated exp flag

### PR DESCRIPTION
**Pull request by rhamzeh**
## Review Checklist
- [ ] I have clicked on ["allow edits by maintainers"](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [ ] I have added documentation for new/changed functionality in this PR or in a PR to [openfga.dev](https://github.com/openfga/openfga.dev) [Provide a link to any relevant PRs in the references section above]
- [ ] The correct base branch is being used, if not `main`
- [ ] I have added tests to validate that the change in functionality is working as expected

### CODE REVIEW

**Review by ewanharris — DISMISSED** [review]

**Review by willvedd — APPROVED** [review]
LGTM. Next time can we add a description, please 🙏 ??

**Review by willvedd — CHANGES_REQUESTED** [review]
Change itself looks good. But can you update the unreleased section in the changelog?

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,6 +8,11 @@ Try to keep listed changes to a concise bulleted list of simple explanations of
 
 ## [Unreleased]
 
+### Changed
+
+* Consistency options has been moved out of experimental and is now enabled by default. The associated experimental option has been removed. [#1889](https://github.com/openfga/openfga/pull/1889)
+  - See [#1764](https://github.com/openfga/openfga/pull/1764) for full details on this feature
```
Comment (willvedd) [inline CHANGELOG.md:14]:
```suggestion
* Consistency options experimental flag has been removed and is now enabled by default. Refer to the [consistency options feature PR](https://github.com/openfga/openfga/pull/1764) for usage details. [#1889](https://github.com/openfga/openfga/pull/1889)
```

**Review by willvedd — APPROVED** [review]
One suggestion. Otherwise, thank you for updating the changelog 🙏

**Review by willvedd — APPROVED** [review]

**Pull request merged**